tests/test_dask_to_root.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+import awkward as ak
+import uproot
+
+skhep_testdata = pytest.importorskip("skhep_testdata")
+dask = pytest.importorskip("dask")
+da = pytest.importorskip("dask.array")
+dask_awkward = pytest.importorskip("dask_awkward")
```

Comment:
This allows the tests to run or be skipped, depending on whether Dask is installed, so you can remove them from the pyproject.toml.